packages/nextjs/hooks/scaffold-eth/useScaffoldContractWrite.ts

```diff
@@ -10,10 +10,9 @@ import { ContractAbi, ContractName, UseScaffoldWriteConfig } from "~~/utils/scaf
 type UpdatedArgs = Parameters<ReturnType<typeof useContractWrite<Abi, string, undefined>>["writeAsync"]>[0];
 
 /**
- * Wrapper for wagmi's useContractWrite hook (with config prepared by usePrepareContractWrite hook)
- * which automatically loads (by name) the contract ABI and address from the deployed contracts
+ * Wrapper around wagmi's useContractWrite hook which automatically loads (by name) the contract ABI and address from the contracts present in deployedContracts.ts & externalContracts.ts
  * @param config - The config settings, including extra wagmi configuration
- * @param config.contractName - deployed contract name
+ * @param config.contractName - contract name
  * @param config.functionName - name of the function to be called
  * @param config.args - arguments for the function
  * @param config.value - value in ETH that will be sent with transaction
```

Comment:
no information about onBlockConfirmation and blockConfirmations,

Reply:
Yup updated at [b9596e5](https://github.com/scaffold-eth/scaffold-eth-2/pull/665/commits/b9596e525b317f67e5fe780228eb4a042374b235) thanks 🙌